buildbot/configure.py

```
@@ -96,10 +96,10 @@
     print("WARNING : The compiler is OTHER")
 
     if not (args.profile == None):
-        raise "can not select a profile with a unknown compiler"
+        raise ValueError("can not select a profile with a unknown compiler")
```

Ruff (0.16.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)